Podfile

```diff
@@ -6,7 +6,7 @@ use_frameworks!
 plugin 'cocoapods-art', :sources => ['onegini']
 
 def oneginiSDKiOS
-   pod 'OneginiSDKiOS', '~> 12.2.0'
+    pod 'OneginiSDKiOS', :git => 'git@github.com:onewelcome/sdk-ios.git', :branch => 'develop'
```

Comment:
This is not working, that's why I'm not able to build the project

Reply:
I've tried hard. The problem is that module map. If this is for developemnt how about putting here sth like 
`pod 'OneginiSDKiOS', :path => 'path/to/local/sdk/repo/on/develop/branch'` ?